.env.local.example

```
@@ -0,0 +1,11 @@
+# Copy to .env.local (gitignored) and fill in. Loaded by `make run-local`.
+
+# proxy
+TWITCH_CLIENT_ID=
+TWITCH_CLIENT_SECRET=
+# Local fallback for /api/magic. Leave MAGIC_LINK_SSM_PARAM unset so it doesn't hit SSM.
+MAGIC_LINK_BLOB=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MAGIC_LINK_BLOB key should go before the TWITCH_CLIENT_ID key

(UnorderedKey)